src/main/java/frc/robot/Config.java

```diff
@@ -44,10 +45,18 @@ public final class Config {
    */
   private static int robotId = -1;
   public static class CANID {
+    public static int PIGEON = robotSpecific(30, 27, 27, 27, 30);
+
     // Arm Subsystem
     public static final int ARM_SPARK_CAN_ID = robotSpecific(5,0,0,0,0,18,18);
     //PCM Can ID 
     public static final int CTRE_PCM_CAN_ID = 1;
+      // Differential Drive CAN IDs
```

Comment:
could put them into class DIFF.

Reply:
All the swerve drive/steer motors canids are in Swerve class.
I think we can separate them into DIFF and SWERVE classes.